test/e2e/kubevirt.go

```diff
@@ -297,7 +297,10 @@ var _ = Describe("Kubevirt Virtual Machines", func() {
 					if utilnet.IsIPv6String(podIP) {
 						port = iperfIPv6Port
 					}
-					output, err := kubevirt.RunCommand(vmi, fmt.Sprintf("iperf3 -t 0 -c %[2]s -p %[3]d --logfile /tmp/%[1]s_%[2]s_%[3]d_iperf3.log &", podName, podIP, port), polling)
+					fileName := fmt.Sprintf("%[1]s_%[2]s_%[3]d_iperf3", podName, podIP, port)
+					// Restart by killing it first, ignore error for first start scenario
+					kubevirt.RunCommand(vmi, fmt.Sprintf("pkill -F /tmp/%s.pid", fileName), polling)
```

Comment:
why do we do this ? It's really not clear (at least to me).

If you can, please move this to a properly named helper so the reader understand it's purpose.